src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1082,7 +1082,8 @@ class ApplicationManagerImpl
     bool operator()(const ApplicationSharedPtr app) const {
       return app
                  ? handle_ == app->device() &&
-                       ProtocolVersion::kV4 == app->protocol_version()
+                       ProtocolVersion::kV4 <= app->protocol_version() &&
```

Comment:
I took it as AppV4 or greater. It looks like it is used to determine what apps support V4 features, so this seems like the way to do it. I would create an AppV5 predicate if one was needed anywhere, but it shouldn't be.